backend/apps/core/middleware/__init__.py

```
@@ -1 +1,110 @@
+                
+            if current == 0:
+                cache.set(cache_key, 1, WINDOW)
+            else:
+                try:
+                    cache.incr(cache_key)
+                except ValueError:
+                    # In case the key expired between get and incr, or is not an integer
+                    cache.set(cache_key, 1, WINDOW)
+                    
+            return True
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)